cms/utils/copy_plugins.py

```diff
@@ -7,20 +7,33 @@ def copy_plugins_to(old_plugins, to_placeholder,
     """
     Copies a list of plugins to a placeholder to a language.
     """
+    # import pdb; pdb.set_trace()
     # TODO: Refactor this and copy_plugins to cleanly separate plugin tree/node
     # copying and remove the need for the mutating parameter old_parent_cache.
     old_parent_cache = {}
-    # For subplugin copy, first plugin's parent must be nulled before copying.
+    # For subplugin copy, top-level plugin's parent must be nulled
+    # before copying.
     if old_plugins:
-        old_plugins[0].parent = old_plugins[0].parent_id = None
-    new_plugins = [old.copy_plugin(to_placeholder, to_language or old.language,
-                                   old_parent_cache, no_signals)
-                   for old in old_plugins]
+        old_parent = old_plugins[0].parent
```

Comment:
No, the point of this is to find all the "old plugins" that are at the top level and set their parent to `None`. We need to consider the first item so we can set its parent to `None` too. Sure, I could move it out of the loop and consider it separately, but then it wouldn't be very DRY.